src/components/ContactForm/FormContainer.jsx

```diff
@@ -5,15 +5,17 @@ import Button from './Button';
 import './FormContainer.css';
 import * as emailjs from 'emailjs-com';
 
+const formContent = {
```

Comment:
It would be better to call this something like `BLANK_FORM`.

Reply:
Though this means undoing the "shortcut" thing I showed you earlier because the value variable name is not longer the same as the key name.

Reply:
Thanks Karen